src/rviz/visualizer_app.cpp

```diff
@@ -164,62 +161,12 @@ bool VisualizerApp::init( int argc, char** argv )
         return false;
       }
 
-      if( vm.count( "in-mc-wrapper" ))
-      {
-        in_mc_wrapper = true;
-      }
-
-      if (vm.count("display-config"))
-      {
-        display_config = vm["display-config"].as<std::string>();
-        if (display_config.size() >= 4 && display_config.substr( display_config.size() - 4, 4 ) == ".vcg")
-        {
-          std::cerr << "ERROR: the config file '" << display_config << "' is a .vcg file, which is the old rviz config format." << std::endl;
-          std::cerr << "       New config files have a .rviz extension and use YAML formatting.  The format changed" << std::endl;
-          std::cerr << "       between Fuerte and Groovy.  There is not (yet) an automated conversion program." << std::endl;
-          return false;
-        }
-      }
-
-      if (vm.count("splash-screen"))
-      {
-        splash_path = vm["splash-screen"].as<std::string>();
-      }
-
-      if (vm.count("help-file"))
-      {
-        help_path = vm["help-file"].as<std::string>();
-      }
-
-      if (vm.count("fixed-frame"))
-      {
-        fixed_frame = vm["fixed-frame"].as<std::string>();
-      }
-
-      if (vm.count("ogre-log"))
-      {
-        enable_ogre_log = true;
-      }
-
-      if (vm.count("no-stereo"))
+      if (display_config.size() >= 4 && display_config.substr( display_config.size() - 4, 4 ) == ".vcg")
       {
-        disable_stereo = true;
-      }
-
-      if (vm.count("opengl"))
-      {
-        //std::cout << vm["opengl"].as<std::string>() << std::endl;
-        force_gl_version = vm["opengl"].as<int>();
-      }
-
-      if (vm.count("disable-anti-aliasing"))
-      {
-        disable_anti_aliasing = true;
-      }
-
-      if (vm.count("verbose"))
-      {
-        verbose = true;
+        std::cerr << "ERROR: the config file '" << display_config << "' is a .vcg file, which is the old rviz config format." << std::endl;
+        std::cerr << "       New config files have a .rviz extension and use YAML formatting.  The format changed" << std::endl;
+        std::cerr << "       between Fuerte and Groovy.  There is not (yet) an automated conversion program." << std::endl;
```

Comment:
> not (yet)
sounds optimistic :wink:

Reply:
Yeah, probably it's time to remove this check completely. I will do this for Noetic.